README.md

```diff
@@ -15,7 +15,22 @@ In a Jupyter notebook (note the ipython magic "[%pip](https://ipython.readthedoc
 %pip install terra-notebook-utils
 ```
 
-To upgrade to the newest version:
+## Upgrading
+
+It is often useful to keep up to date with new features and bug fixes. Installing the latest version of
+terra-notebook-utils depends on your host environment.
+
+From any Jupyter notebook, use the following (and note the leading "%")
+```
+%pip install --upgrade --no-cache-dir terra-notebook-utils
+```
+
+From the CLI on standard Terra notebook runtime, which is available using the Terra terminal button, use
```

Comment:
Maybe include a link to https://github.com/databiosphere/terra-docker#terra-base-images on the phrase: `standard Terra notebook runtime`.

Reply:
added